content/es/docs/concepts/architecture/control-plane-node-communication.md

```diff
@@ -0,0 +1,100 @@
+o [service account tokens](/docs/reference/access-authn-authz/authentication/#service-account-tokens).
+
+Los nodos debe ser aprovisionados con el {{< glossary_tooltip text="certificado" term_id="certificate" >}} raíz público del cluster para que puedan conectar de manera segura al API server junto con credenciales de cliente válidas. Un buen enfoque es que las credenciales del cliente proporcionadas al kubelet estén en forma de un certificado de cliente. Consulta [kubelet TLS bootstrapping] (/docs/reference/access-authn-authz/kubelet-tls-bootstrapping/) para la provisión automatizada de certificados de cliente del kubelet.
+
+{{< glossary_tooltip text="Pods" term_id="pod" >}} que deseen conectarse al servidor API pueden hacerlo de manera segura aprovechando una cuenta de servicio para que Kubernetes inyecte automáticamente el certificado raíz público y un token portador válido en el pod cuando este es instanciado. El servicio  `kubernetes` (en el namespace `default`)  está configurado con una dirección IP virtual que es redirigida (a través de {{< glossary_tooltip text="kube-proxy" term_id="kube-proxy" >}}) al punto final HTTPS en el servidor API.
+
+
+Los componentes del control plane se comunican con el servidor API a través del puerto seguro.
+
+Como resultado, el modo de operación predeterminado para las conexiones desde los nodos y el pod que se ejecutan en los nodos hacia el plano de control está asegurado por defecto y puede funcionar sobre redes no confiables y/o públicas.
+
+## Control plane al nodo
+
+Existen dos formas de comunicación primarias desde el control plane (el servidor API) hacia los nodos. El primero es desde el servidor API hasta el proceso {{< glossary_tooltip text="kubelet" term_id="kubelet" >}} que se ejecuta en cada nodo del clúster. El segundo es desde el servidor API hacia cualquier nodo, pod o servicio a través de la funcionalidad de proxy del servidor API.
+
+
+### Servidor API a kubelet
+
+Las conexiones desde el servidor API al kubelet se utilizan para:
+
+* Obtener registros de los pods.
+* Conectarse (generalmente a través de `kubectl`) a pods en ejecución.
+* Proporcionar la funcionalidad de reenvío de puertos del kubelet.
+
+Estas conexiones terminan en el punto final HTTPS del kubelet. Por defecto, el servidor API no verifica el certificado de servicio del kubelet, lo que hace que la conexión esté sujeta a ataques de intermediario y sea insegura para ejecutarse sobre redes no confiables y/o públicas.
+
+Para verificar esta conexión, usa la bandera `--kubelet-certificate-authority` para proporcionar al servidor API un paquete de certificados raíz para usar en la verificación del certificado de servicio del kubelet.
+
+
+Si eso no es posible, usa [túneles SSH](#ssh-tunnels) entre el servidor API y el kubelet si es necesario para evitar conectarse a través de una red no confiable o pública.
```

Comment:
We should use the same words used in the below translation to have this reference working in this article.

```suggestion
Si eso no es posible, usa [túneles SSH](#tuneles-ssh) entre el servidor API y el kubelet si es necesario para evitar conectarse a través de una red no confiable o pública.
```